prereise/gather/demanddata/nrel_efs/map_states.py

```diff
@@ -9,33 +9,33 @@
 )
 
 
-def decompose_demand_profile_by_state_to_loadzone(agg_dem, save=None):
-    """Transforms the aggregated sectoral demand data so that it is separated by load
-    zone rather than by state.
-
-    :param pandas.DataFrame agg_dem: DataFrame of the aggregated sectoral demand data,
-        where the rows are time steps (in local time) and the columns are the states.
-        This input is intended to be the output of :py:func:`combine_efs_demand`.
+def decompose_demand_profile_by_state_to_loadzone(df, save=None):
+    """Transforms the sectoral demand data so that it is separated by load zone rather
+    than by state.
+
+    :param pandas.DataFrame df: DataFrame of the sectoral demand data, where the rows
+        are time steps (in local time) and the columns are the states. This input is
+        intended to be the output of :py:func:`combine_efs_demand` or the components
+        that are output from :py:func:`partition_flexibility_by_sector`.
     :param str save: Saves a .csv if a str representing a valid file path and file
         name is provided. Defaults to None, indicating that a .csv file should not be
         saved.
-    :return: (*pandas.DataFrame*) -- Aggregate sectoral demand, split by load zone ID.
-    :raises TypeError: if agg_dem is not a pandas.DataFrame or if save is not input as
-        a str.
-    :raises ValueError: if agg_dem does not have the proper time stamps or the correct
-        number of states.
+    :return: (*pandas.DataFrame*) -- Sectoral demand, split by load zone ID.
+    :raises TypeError: if df is not a pandas.DataFrame or if save is not input as a str.
+    :raises ValueError: if df does not have the proper time stamps or the correct number
+        of states.
     """
 
     # Check that a DataFrame is input
-    if not isinstance(agg_dem, pd.DataFrame):
-        raise TypeError("agg_dem must be input as a pandas.DataFrame.")
+    if not isinstance(df, pd.DataFrame):
+        raise TypeError("df must be input as a pandas.DataFrame.")
 
-    # Check the aggregate demand DataFrame time stamps and column headers
-    if not agg_dem.index.equals(
+    # Check the demand DataFrame time stamps and column headers
+    if not df.index.equals(
         pd.date_range("2016-01-01", "2017-01-01", freq="H", closed="left")
     ):
         raise ValueError("This data does not have the proper time stamps.")
```

Comment:
We usually use `timestamp`, i.e., one word

Reply:
Okay, I'll fix that

Reply:
Done.